Environment_Engine/Query/Orientation.cs

```diff
@@ -42,17 +42,58 @@ public static partial class Query
 
         [Description("Returns the orientation of a given environmental object")]
         [Input("environmentObject", "Any object implementing the IEnvironmentObject interface that can have its orientation queried")]
+        [Input("northAngle", "The angle in degrees for north. Default is 90.0.")]
+        [Input("azimuthAngle", "Set to true to return the azimuth angle from north instead of the angle between north and the object normal. Default is false.")]
```

Comment:
The input name on this documentation needs to match the variable name on the input - `returnAzimuthAngle`